.circleci/config.yml

```diff
@@ -97,6 +97,7 @@ jobs:
             sudo pip install nox
       - run:
           name: "Testing Hydra"
+          no_output_timeout: 30m
```

Comment:
I have been trying to keep those down. Why do you have 30 minutes without output?

Reply:
It just takes more than 10 mins to finish the AWS tests :(